### PR TITLE
Feature 392/calculations page

### PR DESCRIPTION
**Pull request by Naira21**
This PR has represented a table to display apartments by choosen house. There is shown info about the apartments, their owner, tariff set for this house and acrued amount of money based on the apartment's square and average tariff

## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/core/api/http-common.ts [hunk 1]
```diff
@@ -2,15 +2,15 @@ import axios, { AxiosResponse } from 'axios';
 import { authToken, notValidRefreshToken, notValidToken } from '@/core/interceptors';
 
 export const HTTP = axios.create({
-  baseURL: 'https://home-project-academy.herokuapp.com/api/0',
```
Comment (yevhendevbox) [inline src/core/api/http-common.ts]: need to change back to https protocol, like it was before (http -> https lines 5 & 13)

File: src/finance/components/CalculationsPage.vue [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+import DataTable from 'primevue/datatable';
+import Dropdown from 'primevue/dropdown';
+import Column from 'primevue/column';
+import Breadcrumb from 'primevue/breadcrumb';
+
+import { StoreModuleEnum } from '@/store/types';
+import { HousesActionsEnum, HousesGettersEnum } from '@/houses/store/types';
+import { CooperationGettersEnum } from '@/cooperation/store/types';
+import { RoutesEnum } from '@/router/types';
+import { TariffModel } from '../models/tariff.model';
```
Comment (yevhendevbox) [inline src/finance/components/CalculationsPage.vue]: just a quick reminder to use '@/...' imports path

File: src/finance/components/CalculationsPage.vue [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+    const updatedApartmentData = () => {
+      const updatedApartment = apartmentsInTheHouse.value.map((apartment: ApartmentModel) => {
+        let owner = 'немає даних';
+        if (apartment.ownerships == undefined) {
+          return;
+        } else {
+          apartment?.ownerships.map((el) => {
+            owner = el.owner.firstName + ' ' + el.owner.lastName + ' ' + el.owner.middleName;
+          });
```
Comment (yevhendevbox) [inline src/finance/components/CalculationsPage.vue]: My suggestion for this conditional:
```js 
   if (!apartment.ownerships) return;
   apartment?.ownership.map((el) => {
 owner = `${el.owner.firstName} ${el.owner.lastName} ${el.owner.middleName}`;  
});
``` 
Just a little less code to wright

**Review by yevhendevbox — DISMISSED** [review]

**Review by anatoliy-stepanchenko — DISMISSED** [review]

File: src/finance/components/CalculationsPage.vue [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+
+    const setHouses = async () => {
+      await store.dispatch(`${StoreModuleEnum.housesStore}/${HousesActionsEnum.SET_HOUSES}`, cooperationId.value);
+      const housesList = await store.getters[`${StoreModuleEnum.housesStore}/${HousesGettersEnum.getHousesData}`];
+      await store.getters[`${StoreModuleEnum.housesStore}/${HousesGettersEnum.getHousesData}`];
+      houses.value = housesList.reduce((acc: any, house: HouseModel) => {
+        return (acc = [
+          ...acc,
+          {
+            adress: `${house.address.city}, ${house.address.street}, ${house.address.houseNumber},
```
Comment (aleonovdementiev) [inline src/finance/components/CalculationsPage.vue]:
```suggestion
            address: `${house.address.city}, ${house.address.street}, ${house.address.houseNumber},
```

**Review by aleonovdementiev — DISMISSED** [review]

**Review by yevhendevbox — APPROVED** [review]

**Review by anatoliy-stepanchenko — APPROVED** [review]

**Pull request merged**